src/integrations/windowsIntegrationManager.ts

```diff
@@ -358,60 +388,65 @@ export default class WindowsIntegrationManager implements IntegrationManager {
     console.log(`kubeconfig integration for ${ distro } set to ${ state }`);
   }
 
-  /**
-   * List the registered WSL2 distributions.
-   */
-  protected get distros(): Promise<string[]> {
+  protected get externalDistros(): Promise<WSLDistro[]> {
```

Comment:
I guess `external` here means "not blacklisted"? But then we also skip over `docker-desktop` / `docker-desktop-data`, which isn't quite external…

Consider having a doc comment.

Reply:
I changed the name to `nonBlacklistedDistros`. I _think_ that is enough to communicate what it does, assuming the reader has knowledge of the blacklist.